CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ include(ExternalProject)
 
 find_package(MLIR REQUIRED CONFIG)
 find_package(AIE REQUIRED)
+set(LibXAIE_ROOT ${Sysroot}/usr ${Sysroot}/opt/xaiengine ${Sysroot}/opt/xaienginev2)
```

Comment:
I don't think these is needed or wanted. The same defaults are set in https://github.com/Xilinx/cmakeModules/blob/master/FindLibXAIE.cmake or the location can be passed in ([for example in the CI](https://github.com/Xilinx/mlir-air/blob/main/.github/workflows/buildAndTest.yml#L87))

Reply:
It seems to me like the default hints compose with CMAKE_SYSROOT... Or at least I couldn't get it to work for me.  I'd prefer to avoid plumbing through explicit paths, which requires us to maintain it in multiple places (acdc, air CI)